packages/terra-navigation-layout/src/NavigationLayout.jsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  withRouter,
+} from 'react-router-dom';
+
+import AppDelegate from 'terra-app-delegate';
+import Layout from 'terra-layout';
+import breakpoints from 'terra-responsive-element/lib/breakpoints.scss';
```

Comment:
We should find a better way to share breakpoints, I'm fine with this for now.

Reply:
Doing so has been on my TODO this for a while; hopefully I can get around to it soon. This is becoming (even more) unsustainable.